Products/CMFPlone/_scripts/_generate_gruntfile.py

```diff
@@ -271,7 +271,7 @@ def resource_to_dir(resource, file_type='.js'):
 # LESS CONFIGURATION
 
 modify_vars = {}
-modify_vars['sitePath'] = '/'
+modify_vars['sitePath'] = "'/'"
```

Comment:
This is in 5.0 but not in 5.1, so I guess this needs forward porting to 5.1 as well.
I don't really know if the change is good; I just compare versions.